sdk/app_cpu1/common/drv/gpio_mux.h

```diff
@@ -12,12 +12,18 @@ typedef enum {
     GPIO_MUX_DEVICE1 = 1,
     GPIO_MUX_DEVICE2 = 2,
     GPIO_MUX_DEVICE3 = 3,
-    GPIO_MUX_DEVICE4 = 4
+    GPIO_MUX_DEVICE4 = 4,
+    GPIO_MUX_DEVICE5 = 5,
+    GPIO_MUX_DEVICE6 = 6,
+    GPIO_MUX_DEVICE7 = 7,
+    GPIO_MUX_DEVICE8 = 8
 } gpio_mux_device_t;
 
 #define GPIO_PORT1 GPIO_MUX_UNUSED
 #define GPIO_PORT2 GPIO_MUX_UNUSED
```

Comment:
What are these used for? Rev E has 4 ports, so I expected more ports here?

Reply:
This is for REV D, not REV E.

I didn't write this, but as far as I can tell, Brandt used these defines in gpio_mux.c to initialize both ports to "GPIO_MUX_UNUSED", or 0.

https://github.com/Severson-Group/AMDC-Firmware/blob/ddd15424e045db503d0cc9b94757ae313616b551/sdk/app_cpu1/common/drv/gpio_mux.c?plain=1#L9-L13

This is documented [here](https://github.com/Severson-Group/AMDC-Firmware/blob/gpio-direct/sdk/app_cpu1/common/drv/docs/GPIO-Mux.md#macros) in GPIO-Mux.md

Reply:
@codecubepi good find on these doc pages. I forget they existed.

I'll make an issue in docs.amdc.dev repo to move these to the website.

Reply:
Done: https://github.com/Severson-Group/docs.amdc.dev/issues/82